spec/coffee/directives/api/utils/_async.spec.coffee

```diff
@@ -2,21 +2,23 @@ describe "_async", ->
   rootScope = null
   timeout = null
   q = null
+  browser = null
 
   digest = (fn, times = 1) =>
     fn()
     if times
       _.range(times).forEach -> # i would like to say that it sucks that I have to do this.. (angular)
-        timeout?.flush()
+        timeout?.flush() if browser.deferredFns?.length
```

Comment:
The reason I added the checks for deferredFns was because timeout, rootScope are actually being put into the global space for tests. So somehow Circle's tests were causing _async's tests to fail. So yes there needs to be a second PR to fix digest , timeout, and rootScope to not be global.